src/main/java/org/traccar/protocol/Gt06ProtocolDecoder.java

```diff
@@ -794,22 +794,34 @@ private Object decodeBasic(Channel channel, SocketAddress remoteAddress, ByteBuf
 
             position.set(Position.KEY_TYPE, type);
 
+            if (type == MSG_GPS_LBS_2 && "NT20".equals(model)) {
+                buf.readUnsignedByte(); // location source type
+                buf.skipBytes(8); // imei
+
+                DateBuilder dateBuilder = new DateBuilder((TimeZone) deviceSession.get(DeviceSession.KEY_TIMEZONE))
+                        .setDate(buf.readUnsignedByte(), buf.readUnsignedByte(), buf.readUnsignedByte())
+                        .setTime(buf.readUnsignedByte(), buf.readUnsignedByte(), buf.readUnsignedByte());
```

Comment:
To avoid duplication, I suggest we extract this into a method that returns a `Date` object.